dev_scripts/find_images_to_rebuild.py

```diff
@@ -40,15 +46,30 @@
 
 errata = Errata()
 kwargs = {}
-if container_images:
+
+if args.container_img:
     EventClass = ManualRebuildWithAdvisoryEvent
-    kwargs['container_images'] = container_images
+    kwargs['container_images'] = args.container_img
 else:
     EventClass = ErrataAdvisoryStateChangedEvent
+
+event_id = None
+if args.cassette:
+    cassette_name = os.path.splitext(os.path.basename(args.cassette))[0]
```

Comment:
Optional: You could verify if the extension is `.yml` to avoid someone having a cassette that has the extension `.yaml` and have it query Lightblue directly.